avromatic.gemspec

```diff
@@ -31,8 +31,8 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.7'
 
-  spec.add_runtime_dependency 'activemodel', '>= 5.2', '< 7.1'
-  spec.add_runtime_dependency 'activesupport', '>= 5.2', '< 7.1'
+  spec.add_runtime_dependency 'activemodel', '>= 5.2', '< 7.2'
+  spec.add_runtime_dependency 'activesupport', '>= 5.2', '< 7.2'
   spec.add_runtime_dependency 'avro', '>= 1.10.0', '< 1.12'
```

Comment:
Can we drop support for avro 1.10.x if we're making a breaking change anyway?

Reply:
@jturkel done